src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_upd_routines.F90

```diff
@@ -4923,7 +5202,7 @@ subroutine check_compact_support(                                &
     
     select case (update_type)
 
-    case (1,3,4,5,6,9)   ! "1d" updates
+    case (1,3,4,5,6,9,11)   ! "1d" updates
```

Comment:
The implementation of the snow cover analysis is based on the "1d filter" cases.  These cases worked in LDASsa up until the time when Clara used them in her MWR paper, which uses code from about 2017, well before GEOSldas came online.  It is possible that the old 1d filters no longer work in GEOSldas.  For GEOSldas development, we routinely tested and verified only the 3d Tb analysis of the SMAP L4_SM system.  That is, the floating point issue that @jpark50 is experiencing may be related to something that is not specific to the snow cover analysis.